src/main/java/controller/Controller.java

```diff
@@ -0,0 +1,11 @@
+package controller;
+
+import webserver.Request;
+import webserver.Response;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+public interface Controller {
+    Response serving(Request request) throws IOException, URISyntaxException;
```

Comment:
메서드명은 동사로 작성하는게 좋습니다.

```suggestion
    Response service(Request request) throws IOException, URISyntaxException;
```